pixeltable/metadata/__init__.py

```diff
@@ -55,6 +55,11 @@ def upgrade_md(engine: sql.engine.Engine) -> None:
         system_info = session.query(SystemInfo).one().md
         md_version = system_info['schema_version']
         assert isinstance(md_version, int)
+        if md_version > VERSION:
+            raise RuntimeError(
```

Comment:
We use `pixeltable.excs.Error` for all exceptions that carry user-facing error messages.